### PR TITLE
confile: add lxc.namespace.share.[namespace], lxc.namespace.keep, lxc.namespace.clone

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

**Comment by brauner** [comment]
@flx42, @3XX0

File: src/lxc/start.c [hunk 1]
```diff
@@ -1567,7 +1554,8 @@ static int lxc_spawn(struct lxc_handler *handler)
 	 * again.
 	 */
 	if (wants_to_map_ids) {
-		if (!handler->conf->ns_share[LXC_NS_USER]) {
+		if (!handler->conf->ns_share[LXC_NS_USER] ||
+		    (handler->conf->ns_keep & CLONE_NEWUSER) > 0) {
```
Comment (hallyn) [inline src/lxc/start.c]: Shouldn't this be

if (!handler->conf->ns_share[LXC_NS_USER] &&
     (handler->conf->ns_keep & CLONE_NEWUSER) == 0) {
?

Comment (brauner) [reply src/lxc/start.c]: Could catch, thanks!

Comment (brauner) [reply src/lxc/start.c]: s/Could/Good/

**Comment by brauner** [comment]
@hallyn, this is good to go.

**Pull request merged**